executions/graphql-kotlin-transaction-loader-instrumentation/README.md

```diff
@@ -1,49 +1,49 @@
-# GraphQL Kotlin Transaction Batcher Instrumentation
+# GraphQL Kotlin Transaction Loader Instrumentation
```

Comment:
for consistency -> should we rename this module to `graphql-kotlin-dataloader-instrumentation`?